formats/financial_releases_geoblocker/frontend/examples/financial_releases_geoblocker.json

```diff
@@ -1,5 +1,5 @@
 {
-  "base_path": "/financial-releases-geoblocker",
```

Comment:
I think we need to update this to `/lloydsshares/location` because it's an example format.